test/hierarchical-data.test.js

```diff
@@ -12,7 +12,7 @@ describe('cap/samples - Hierarchical Data', ()=>{
 	`
 	const model = cds.compile.for.nodejs(csn)
 	const {Categories:Cats} = model.definitions
-	const {expect} = cds.test
+	const { expect } = cds.test.in(__dirname,'..')
```

Comment:
To me that seems like a fair workaround.
I see it as a workaround as we typically require a common structure for cds.test which we do not have in these tests. Hence, we can use the top level package.json as source.